app/views/reservations/edit.html.haml

```diff
@@ -25,6 +25,13 @@
 
   = render "reservation_fields", f: f
 
+  .well
```

Comment:
Ah okay, I had moved it because it looked really weird especially before adding the extra label below, but the consistency makes sense